src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+
+        // Check if the user with the given email exists
+        if (user != null) {
+            // Check if the provided password matches the user's password
+            return user.getPassword().trim().equals(password.trim());
+        }
+
         return false;
```

Comment:
good job, we can also combine these two conditions and return one logical expression instead. Also, trimming is a good idea, but on a service level we usually receive sanitized input and check more complex business rules, so you can omit it in this task